hotsos/defs/scenarios/storage/nfs/nfs-detect-mount-name-resolution-failure.yaml

```diff
@@ -0,0 +1,15 @@
+checks:
+  has_nfs_name_resolution_failures:
+    input:
+      path: ['var/log/syslog', 'var/log/boot.log']
+    expr: '.*mount.nfs.*Failed to resolve server (.*): (Name or service not known|Temporary failure in name resolution).*'
+conclusions:
+  nfs_name_resolution_failures:
+    decision: has_nfs_name_resolution_failures
+    raises:
+      type: NFSNameResolutionError
+      message: >-
+        Name resolution failures have been detected for the following NFS mounts:
+        {names}
```

Comment:
I think it would be nice to go ahead and make some suggestions for next steps in terms of things to check and actions to take but we can leave that for a second patch of you like.

Reply:
Well, that I don't know of, really. I saw the issue in the queue and decided to have a go at it. I'm not an expert in NFS 😅